addon/styles/addon.css

```diff
@@ -26,14 +26,14 @@
     padding: 0.2em 1.5em 0em;
     position: absolute;
     white-space: nowrap;
-    top: 2px;
+    top: 0;
```

Comment:
Did this change because something else made it change, or is this a matter of opinion?

Reply:
@ef4 reversed this change.